File: terraMarket/src/main/java/com/example/demo/repository/entity/Tienda.java
package com.example.demo.repository.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name = "tiendas")
public class Tienda {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String nombre;
    private String logo;
    private String descripcion;
    private String imagen;
    private boolean activo;


}

File: Front/terraMarket/src/app/models/tienda.ts
import { CategoriaT } from './categoriaT';
import { Mercado } from "./mercado";

export interface Tienda {
  id: number;
  nombre: string;
  logo: string;
  descripcion: string;
  imagen: string;
  activo: boolean;
  categoriaT: CategoriaT[];
}
/*    private Long id;
    private String nombre;
    private String logo;
    private String descripcion;
    private String imagen;
    private boolean activo;
    private MercadoDTO mercadoDTO;*/
